tools/osemosys/features.yaml

```diff
@@ -0,0 +1,401 @@
+      source: 
+        - https://otoole.readthedocs.io/en/latest/
+      value: y
+  optimisation_problem:
+    configuration:
+      source: []
+      value: n
+    decomposition:
+      source: []
+      value: n
+    near_optimal:
+      source: []
+      value: n
+    sim + opt:
+      source: []
+      value: n
+    user_defined_constraints:
+      source:
+        - https://github.com/OSeMOSYS/OSeMOSYS_GNU_MathProg/pull/75
+      value: dev
+    user_defined_variables:
+      source: []
+      value: n
+  postprocessing:
+    aggregation:
+      source: []
+      value: n
+    curtailment:
+      source: []
+      value: n
+    financial:
+      source: []
+      value: n
+    future_cost_function:
+      source: []
+      value: n
+    gtc:
+      source: []
+      value: n
+    impact_assessment:
+      source: []
+      value: n
+    lcoe:
+      source: []
+      value: n
+    lodf:
+      source: []
+      value: n
+    ptdf:
+      source: []
+      value: n
+    shadow_prices:
+      source: []
+      value: n
+  preprocessing:
+    download:
+      source: []
+      value: n
+    forecasting:
+      source: []
+      value: n
+    spatial_distribution:
+      source: []
+      value: n
+  resource_adequacy:
+    eens:
+      source: []
+      value: n
+    lolp:
+      source: []
+      value: n
+    maintenance_scheduling:
+      source: []
+      value: n
+    monte_carlo:
+      source:
+        - https://github.com/KTH-dESA/esom_gsa
+        - https://open-research-europe.ec.europa.eu/articles/3-30/v1
+      value: dev
+    sampling:
+      source:
+        - https://github.com/KTH-dESA/esom_gsa
+        - https://open-research-europe.ec.europa.eu/articles/3-30/v1
+      value: dev
+  robustness:
+    asset_outages:
+      source: []
+      value: n
+    eens_limit:
+      source:
+        - https://osemosys.readthedocs.io/en/latest/manual/Advanced%20functionalities.html#stochastic-modelling-of-energy-security-assessment
+      value: dev
+    lolp_limit:
+      source: []
+      value: n
+    multi_stage_stochastic:
+      source: []
+      value: n
+    ram:
+      source: []
+      value: n
+    reserve_limit:
+      source:
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#reserve-margin
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#reserve-margin-constraints
+      value: y
+    two_stage_stochastic:
+      source: []
+      value: n
+    voll:
+      source: []
+      value: n
+  spatial__resolution__simplification:
+    network:
+      source: []
+      value: n
+    resampling:
+      source: 
+        - https://github.com/KTH-dESA/GSA_Spatial_temporal
+        - https://doi.org/10.1016/j.isci.2025.111792
+      value: dev
+  system__carriers:
+    coupled:
+      source:
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#sets
+      value: y
+    decoupled:
+      source: []
+      value: n
```

Comment:
I guess that by definition, if `coupled` is possible, so is `decoupled`, so you could update this to `y`. I might remove `decoupled` and assume that it is the baseline and definition of distinct energy carriers is the "feature"